src/components/LinkCard.jsx

```diff
@@ -0,0 +1,20 @@
+import formatDate from '../utils/formatDate';
+import diffTime from '../utils/diffTime';
+import styles from '../css/Card.module.css'
+
+function LinkCard({item}) {
```

Comment:
```suggestion
function LinkCard({item : {url, id, description ,,,,}}) {
```
2중 구조분해할당 해주시면 아래 코드가 간결해질 거 같아요!